src/AppKit/NSFont.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Runtime.InteropServices;
+using XamCore.CoreFoundation;
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+using XamCore.CoreGraphics;
+using XamCore.CoreAnimation;
+using XamCore.CoreText;
+
+using CGGlyph = System.UInt16;
+
+namespace XamCore.AppKit {
+	public partial class NSFont
+	{
+		public static NSFont FromCTFont (CTFont font)
+		{
+			if (font == null)
+				return null;
+			return new NSFont (font.Handle);
+		}
+
+		[Mac (10,13)]
+		public unsafe CGRect [] GetBoundingRects (CGGlyph [] glyphs)
+		{
+			if (glyphs == null)
+				throw new ArgumentNullException ("glyphs");
+			if (glyphs.Length < 1)
+				throw new ArgumentException ("glyphs array is empty");
+
+			CGRect [] bounds = new CGRect [glyphs.Length]; 
+			fixed (CGRect* boundsPtr = &bounds [0]) {
+				fixed (CGGlyph* glyphsPtr = &glyphs [0]) {
+					_GetBoundingRects ((IntPtr)boundsPtr, (IntPtr)glyphsPtr, (nuint)glyphs.Length);
```

Comment:
We need to test what happens with an empty (non null) array since that might return something valid.
If that's the case then `&glyphs [0]` is invalid.